aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -41,22 +41,21 @@ void expm1_kernel_cuda(TensorIterator& iter) {
   });
 }
 
-// We manually overload rsqrt because std::rsqrt does not work with thrust::complex types.
+// We manually overload rsqrt because std::rsqrt does not work with complex types.
 template<typename scalar_t>
 __host__ __device__ static inline scalar_t rsqrt_wrapper(scalar_t v) {
   return ::rsqrt(v);
 }
 
 template<typename T>
-__host__ __device__ static inline thrust::complex<T> rsqrt_wrapper(thrust::complex<T> v) {
-  const thrust::complex<T> one = thrust::complex<T>(1.0, 0);
-  return one/thrust::sqrt(v);
+__host__ __device__ static inline c10::complex<T> rsqrt_wrapper(c10::complex<T> v) {
+  const c10::complex<T> one = c10::complex<T>(1.0, 0);
+  return one / std::sqrt(v);
```

Comment:
just to confirm -- this uses the function defined in std namespace for c10::complex right? might be worth adding a note

Reply:
Yes, and comment added!